src/pylon/src/nginx.conf.template

```diff
@@ -114,8 +114,13 @@ http {
     }
 
     # Prometheus API server.
-    location ~ ^/prometheus/api(.*)$ {
-      proxy_pass {{PROMETHEUS_URI}}/api$1$is_args$args;
+    location ~ ^/prometheus(.*)$ {
+      proxy_pass {{PROMETHEUS_URI}}/prometheus$1$is_args$args;
+    }
+
+    # alert-manager server.
+    location ~ ^/alert-manager(.*)$ {
+      proxy_pass {{ALERT_MANGER_URI}}/alert-manager$1$is_args$args;
```

Comment:
Does web pages of alert-manager need links replacement like webportral?

Reply:
I think no, but not sure.